src/Views/DogAdmin.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { addDog } from '../services/DogRoute';
+import DogForm from '../Components/DogCard/DogForm';
+
+export default function CreateDog() {
+  const [name, setName] = useState('');
+  const [bio, setBio] = useState('');
+  const [image, setImage] = useState('');
+  const [age, setAge] = useState('');
+  const [breed, setBreed] = useState('');
+
+  const handleSubmit = async (e) => {
+    try {
+      alert('DOG HAS BEEN ADDED');
+    } catch {
+      alert('DOG WAS NOT ADDED');
+    }
+    e.preventDefault();
+    setAge(''), setName(''), setBio(''), setBreed(''), setImage('');
+    await addDog(name, bio, image, age, breed);
```

Comment:
This This await addDog needs to be in the try block
try {
await addDog...
alert...
}